Content.Server/GameObjects/Components/GUI/InventoryComponent.cs

```diff
@@ -33,9 +33,13 @@ public class InventoryComponent : SharedInventoryComponent, IExAct, IEffectBlock
 #pragma warning restore 649
 
         [ViewVariables]
-        private readonly Dictionary<Slots, ContainerSlot> SlotContainers = new Dictionary<Slots, ContainerSlot>();
+        private readonly Dictionary<Slots, ContainerSlot> _slotContainers = new Dictionary<Slots, ContainerSlot>();
 
-        private KeyValuePair<Slots, (EntityUid entity, bool fits)>? HoverEntity;
+        private KeyValuePair<Slots, (EntityUid entity, bool fits)>? _hoverEntity;
+
+        public IReadOnlyDictionary<Slots, ContainerSlot> SlotContainers => _slotContainers;
+
+        public event Action OnChanged;
```

Comment:
It would probably be better to rename this to OnItemChanged, to differentiate it from the active hand changing.

Reply:
Done